src/main/java/com/thevoxelbox/voxelsniper/brush/type/entity/EntityRemovalBrush.java

```diff
@@ -29,7 +30,10 @@ public EntityRemovalBrush() {
     public void handleCommand(String[] parameters, Snipe snipe) {
         SnipeMessenger messenger = snipe.createMessenger();
         for (String currentParam : parameters) {
-            if (!currentParam.isEmpty() && (currentParam.charAt(0) == '+' || currentParam.charAt(0) == '-')) {
```

Comment:
Not validating the input is quite dangling. Looking at the code `/b entityremoval +#` would display an empty space in the chat e.g. or allow any kind of typo to screw up.